tests/config/nf-test.config

```diff
@@ -0,0 +1,50 @@
+params {
+    publish_dir_mode = "copy"
+    singularity_pull_docker_container = false
+    test_data_base = 'https://raw.githubusercontent.com/nf-core/test-datasets/modules'
+}
+
+process {
+    cpus = 2
+    memory = 3.GB
+    time = 2.h
+}
+
+profiles {
+    singularity {
+        singularity.enabled = true
+        singularity.autoMounts = true
+    }
+    conda {
+        conda.enabled = true
+    }
+    mamba {
+        conda.enabled  = true
+        conda.useMamba = true
+    }
+    podman {
+        podman.enabled = true
+        podman.userEmulation = true
```

Comment:
nf-core removed this configuration option recently https://github.com/nf-core/modules/pull/4626

Reply:
good catch. I updated the tests/config/nf-test.conf and tests/config/nextflow.config to remove the <docker|podman>.userEmulation argument.